recipes/pitot/license.txt

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
You can remove this license file since it appears to be included in the source distribution.